#!/usr/bin/env python
PACKAGE = "control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("P_roll", double_t, "Proportional gain for roll angle.", 1.0)
gen.add("D_roll", double_t, "Derivative gain for roll angle", 0.1)
gen.add("P_pitch", double_t "Proportional gain for pitch angle.", 1.0)
gen.add("D_pitch", double_t, "Derivative gain for pitch angle.", 0.1)
gen.add("P_yaw", double_t, "Proportional gain for yaw angle.", 1.0)
gen.add("D_yaw", double_t, "Derivative gain for yaw angle.", 0.1)

exit(gen.generate(PACKAGE, "control", "GeometricControl"))
